web/src/OverviewSidebarOptions.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import styled from "styled-components"
+import { SidebarOptions } from "./types"
+
+const SidebarOptionsSetterStyle = styled.div`
+  display: flex;
+`
+
+type SidebarResourceTypeFilterProps = {
+  curState: SidebarOptions
+  toggleShowResources: () => void
+  toggleShowTests: () => void
+}
+
+export function SidebarOptionsSetter(
```

Comment:
our standard naming convention on this would be something like

export default function OverviewSidebarOptions(props: OverviewSidebarOptionsProps) {
  return <OverviewSidebarOptionsRoot>...</OverviewSidebarOptionsRoot>
}

for stuff like this, a good rule of thumb is to have a consistent naming convention even at the expense of an accurate name

I count at least 4 conflicting names in this file, which seems not ideal for a single component

Reply:
ah whoops at least some of that is stuff i missed during a rename 😅  good catch